tests/src/test/java/alluxio/underfs/UnderStorageSystemInterfaceIntegrationTest.java

```diff
@@ -143,6 +143,27 @@ public void createOpenLarge() throws IOException {
   }
 
   /**
+   * Tests seek.
+   */
+  @Test
+  public void createOpenSeek() throws IOException {
+    String testFile = PathUtils.concatPath(mUnderfsAddress, "testFile");
+    OutputStream outputStream = mUfs.create(testFile);
+    int numBytes = 100;
```

Comment:
nit: how long would this test case take with 100 open/seek/close iterations? especially with slower `-Ps3test`. How about reducing to 10?